src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/jpa/Address.kt

```diff
@@ -118,6 +120,10 @@ abstract class Address(
   )
   open var country: Country? = null,
 
+  @OneToMany(mappedBy = "id.address", cascade = [CascadeType.ALL], fetch = LAZY, orphanRemoval = true)
+  @OrderBy("id.usageCode")
+  val usages: MutableList<AddressUsage> = mutableListOf(),
+
   @Column(name = "VALIDATED_PAF_FLAG")
   @Convert(converter = YesNoConverter::class)
   open var validatedPAF: Boolean = false,
```

Comment:
if validatedPAF is irrelevant (sorry, not part of this PR) - shall we remove it?